config/initializers/simple_form.rb

```diff
@@ -0,0 +1,214 @@
+    # given input by passing: `f.input EXTENSION_NAME => false`.
+    # You can make any of these extensions optional by
+    # renaming `b.use` to `b.optional`.
+
+    # Determines whether to use HTML5 (:email, :url, ...)
+    # and required attributes
+    b.use :html5
+
+    # Calculates placeholders automatically from I18n
+    # You can also pass a string as f.input placeholder: "Placeholder"
+    b.use :placeholder
+
+    ## Optional extensions
+    # They are disabled unless you pass `f.input EXTENSION_NAME => true`
+    # to the input. If so, they will retrieve the values from the model
+    # if any exists. If you want to enable any of those
+    # extensions by default, you can change `b.optional` to `b.use`.
+
+    # Calculates maxlength from length validations for string inputs
+    # and/or database column lengths
+    b.optional :maxlength
+
+    # Calculate minlength from length validations for string inputs
+    b.optional :minlength
+
+    # Calculates pattern from format validations for string inputs
+    b.optional :pattern
+
+    # Calculates min and max from length validations for numeric inputs
+    b.optional :min_max
+
+    # Calculates readonly automatically from readonly attributes
+    b.optional :readonly
+
+    ## Inputs
+    # b.use :input, class: 'input', error_class: 'is-invalid', valid_class: 'is-valid'
+    b.use :label_input
+    b.use :hint,  wrap_with: { tag: :span, class: :hint }
+    b.use :error, wrap_with: { tag: :span, class: :error }
+
+    ## full_messages_for
+    # If you want to display the full error message for the attribute, you can
+    # use the component :full_error, like:
+    #
+    # b.use :full_error, wrap_with: { tag: :span, class: :error }
+  end
+
+  config.wrappers :govuk, tag: "div", class: "govuk-form-group", error_class: "has-error" do |b|
+    b.use :html5
+    b.use :placeholder
+    b.use :label, class: "govuk-label"
+    b.use :input, class: "govuk-input"
+    b.use :error, wrap_with: { tag: "span", class: "help-inline govuk-error-message" }
+    b.use :hint,  wrap_with: { tag: "p", class: "help-block" }
+  end
+
+  config.wrappers :govuk_select, tag: "div", class: "govuk-form-group", error_class: "has-error" do |b|
+    b.use :html5
+    b.use :placeholder
+    b.use :label, class: "govuk-label"
+    b.use :input, class: "govuk-select"
+    b.use :error, wrap_with: { tag: "span", class: "help-inline govuk-error-message" }
+    b.use :hint,  wrap_with: { tag: "p", class: "help-block" }
+  end
+
+  config.wrappers :boolean_input, tag: 'div', class: "govuk-checkboxes" do |b|
+    b.use :html5
+    b.wrapper tag: "div", class: 'govuk-checkboxes__item', error_class: 'has-error' do |ba|
+      ba.use :input, class: "govuk-checkboxes__input"
+      ba.use :label, class: "govuk-label govuk-checkboxes__label"
+      ba.use :error, wrap_with: { tag: "span", class: "help-inline govuk-error-message" }
+      ba.use :hint,  wrap_with: { tag: "p", class: "help-block" }
+    end
+  end
```

Comment:
Variation of the main govuk wrapper to handle the alternative arrangement needed for boolean elements (check boxes).